docs/docs/resources/pipeline_config/config.yaml

```diff
@@ -0,0 +1,55 @@
+# CONFIGURATION
+#
+# The path to the folder containing the defaults file and the environment
+# defaults files.
+# REQUIRED
+defaults_path: /path/to/defaults
+# The environment you want to generate and deploy the pipeline to. Suffix your
+# environment files with this value (e.g. defaults_development.yaml for
+# environment=development).
+# REQUIRED
+environment: development
+# The kafka broker address.
+# REQUIRED
+broker: "http://localhost:9092"
+# The name of the defaults file and the prefix of the defaults environment file.
+defaults_filename_prefix: defaults
+# Base class for settings, allowing values to be overridden by environment
+# variables. This is useful in production for secrets you do not wish to save in
+# code, it plays nicely with docker(-compose), Heroku and any 12 factor app
+# design.
+topic_name_config: 
+  # Configures the value for the variable ${output_topic_name}
+  default_output_topic_name: ${pipeline_name}-${component_name}
+  # Configures the value for the variable ${error_topic_name}
+  default_error_topic_name: ${pipeline_name}-${component_name}-error
+# Address to the rest proxy REST API.
+kafka_rest_host: "http://localhost:8082"
+# Address to the kafka connect REST API.
+kafka_connect_host: "http://localhost:8083"
+# The timeout in seconds that specifies when actions like deletion or deploy
+# timeout.
+timeout: 300
+# Flag for `helm upgrade --install`.
+# Create the release namespace if not present.
+create_namespace: false
```

Comment:
weird, why is this here and not under `helm_config`

Reply:
Yeah, it's a bit weird

Reply:
https://github.com/bakdata/kpops/pull/138#discussion_r1124262751